aries-site/src/utils/pageVisitTracker.js

```diff
@@ -0,0 +1,29 @@
+//this is the function i call to report whether or not the update should be shown given the most recent commit and last-visit records
```

Comment:
Can you refine the comments in this file a little to be more "neutral"/3rd person?

For example:
```suggestion
// Determine whether or not the update should be shown given the most recent commit and user visitation
```